liquidator/liquidator.js

```diff
@@ -43,7 +43,7 @@ class Liquidator {
 
     // Set and validate config settings
     Object.keys(defaultConfig).forEach(field => {
-      this[field] = config && config[field] ? config[field] : defaultConfig[field].value;
+      this[field] = config && Object.keys(config).includes(field) ? config[field] : defaultConfig[field].value;
```

Comment:
I realize I needed to change this in case `config[field] === 0`, for example if the `disputeDelay = 0`, which is a perfectly reasonable value. The problem is if the `value === 0` then this boolean returns false un expectedly even though its a valid value.

Reply:
My suggestion is to use the `in` operator for the cleanest way to check if a field is provided.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/in

Reply:
nice I'll make that change